site/formulas/index.md

```diff
@@ -20,11 +20,28 @@ dangerous and unsafe to implement the Ristretto functions as operating on
 arbitrary curve points, rather than only on the representatives contained in a
 distinct Ristretto point type.
 
-## TODO
-
-This section should have explicit formulas for Ristretto, aimed at
-implementations.
-
-It should also have a description of "how to implement Ristretto", i.e.,
-what functionality is required: encoding, decoding, equality,
-hash-to-point, type safety, etc.
+## Implementation
+
+The Decaf paper suggests using a quotient group, such as \\( \mathcal E /
+\mathcal E[4] \\) or \\( 2 \mathcal E[2] \\), to implement a prime-order group
+using a non-prime-order curve with cofactor \\( 4 \\).
```

Comment:
I think that this text doesn't really fit, since the text right above it is already talking about Ristretto points as being represented by Edwards points, and the "What is Ristretto" section already covers this material.

I think that instead it would be better to give an explicit list (i.e., something checklistable) of what's required to implement Ristretto:

- [ ] a distinct type for Ristretto points, which forwards curve operations to their representatives

- [ ] the Ristretto encoding function

- [ ] the Ristretto decoding function

- [ ] the Ristretto equality check

- [ ] the Ristretto hash-to-group function

Reply:
Should be fixed in f479893